### PR TITLE
fix: display nft without name in bsx

### LINKED ISSUE #3199: Missing links to NFTs in Collection/holders
Seems like the under holder missing a link to NFT?

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3199
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

It will display id of nft if the nft does not have a name.

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Review by petersopko — CHANGES_REQUESTED** [review]
Links here are still pointing to rmrk
image

**Review by petersopko — CHANGES_REQUESTED** [review]
some of them have name though?
image

File: components/rmrk/Gallery/Holder/Holder.vue [hunk 1]
```diff
@@ -145,7 +145,7 @@
                     name: 'rmrk-gallery-id',
```
Comment (vikiival) [inline components/rmrk/Gallery/Holder/Holder.vue:145]: bad url param

**Review by vikiival — CHANGES_REQUESTED** [review]

**Comment by Jarsen136** [comment]
> some of them have name though? 

ref: https://github.com/kodadot/nft-gallery/issues/3199#issuecomment-1160098542

**Review by petersopko — APPROVED** [review]
wfm, the name part will be solved in https://github.com/kodadot/packages/issues/17

**Review by petersopko — APPROVED** [review]

**Comment by petersopko** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 53.32 USD/KSM ~ 1.875 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xa5c2a2da8b00402664c623269e109e10a18ad25067e3a81c8a53c8fca8a63120](https://kusama.subscan.io/extrinsic/0xa5c2a2da8b00402664c623269e109e10a18ad25067e3a81c8a53c8fca8a63120)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**